crates/viewer/re_space_view_spatial/src/contexts/transform_context.rs

```diff
@@ -312,18 +312,23 @@ fn get_parent_from_child_transform(
         entity_path,
         [
             Transform3D::name(),
-            TransformMat3x3::name(),
             Translation3D::name(),
+            Scale3D::name(),
+            TransformMat3x3::name(),
         ],
     );
     if result.components.is_empty() {
         return None;
     }
 
+    // Order is specified by order of components in the Transform3D archetype.
```

Comment:
We could add a `debug_assert` here to check that the fields in the archetype matches what we expect by the code below, by using the reflection API (`ArchetypeReflection::fields`)